packages/sources/bitgo-reserves/src/config/index.ts

```diff
@@ -6,4 +6,14 @@ export const config = new AdapterConfig({
     type: 'string',
     default: 'https://reserves.usdstandard-test.com/por.json',
   },
+  API_ENDPOINT2: {
```

Comment:
What's the difference between API_ENDPOINT and API_ENDPOINT2 (same for reserves and reserves2)?

Trying to find a better name to distinguish when to use one endpoint vs the other

Reply:
replaced the original reserves endpoint now